### DISCUSSION TITLE
Can you edit and redeploy older versions of docs with mike?

### DISCUSSION THREAD

**Discussion by paulinestacey** [jimporter/mike#216] [General]
Hi, docs quote "mike is built around the idea that once you've generated your docs for a particular version, you should never need to touch that version again" - however, I need the flexibility of being about to edit older versions, if needed. Please can someone confirm if mike plugin allows older particular versions to be edited? Thanks.

**Comment by jimporter** [comment]
Yes, you can. You probably *shouldn't* update docs for an already-released version, but it works just fine, and you can use this e.g., for deploying a snapshot of the latest in-development docs.
